eng/pipelines/templates/steps/analyze.yml

```diff
@@ -59,6 +59,11 @@ steps:
     condition: and(succeededOrFailed(), ne(variables['Build.Reason'],'PullRequest'))
     displayName: 'Component Detection'
 
+  - pwsh: |
+      Get-ChildItem $(Build.SourcesDirectory)/sdk/${{parameters.ServiceDirectory}} -Filter *.md -Recurse `
+        | eng/scripts/Find-InvalidSlug.ps1 -IncludeParents
```

Comment:
This should go into eng/common if we're going to use it across language repos. The step should too.

Reply:
I updated https://gist.github.com/heaths/08a395af1800a2826be8b8759e735394 with some improvements (no change in desired behavior), including Write-Error instead of Write-Warning. File info is only passed back now if you pass `-PassThru` - not exactly recommended with the `Find` standard verb, but 🤷‍♂️.

Reply:
I'll start a PR. If we're discussing adding more validation, I'm going to rename this to be less-specific to slugs and more general, e.g. `Test-SampleMetadata.ps1`.

Reply:
See https://github.com/Azure/azure-sdk-tools/pull/1555

Reply:
Azure/azure-sdk-tools#1555 has been merged.